docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+
+{{< figure src="/media/docs/tempo/screenshot-grafana-tempo-span-filters-v10-1.png" max-width="750px" caption="Traces span filtering" >}}
+
+### Configuration page redesign for Loki and Elasticsearch
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+The Loki and Elasticsearch data source configuration pages have been redesigned to make getting started and setting up data sources as simple and easy to understand as possible. You can now find new subsections with links to configuration pages, as well as tooltips to assist you with configuring and customizing data sources.
+
+### Loki query splitting
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+In response to different query performance scenarios, we've implemented query splitting. Now, queries that request more than a day of data are split into sub-requests, each with a duration of one day. For example, requesting seven days of logs will produce seven requests of one day.
+
+### Easier to use query editor for Elasticsearch
+
+<!-- Gabor Farkas -->
+
+_Generally available in all editions of Grafana_
+
+The Elasticsearch query editor now allows convenient switching between logs, metrics, and raw data directly from the top, eliminating the need to go through the metric selector.
+
+### Metrics explorer
+
+The Metrics explorer is a new feature that enhances metrics browsing in the Prometheus query builder. The Metrics explorer makes it easier for you to find the right metric, and get comfortable with PromQL. You can now explore metrics with additional metadata, perform fuzzy search on the metric name or description, and filter on the Prometheus type.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-metrics-explorer.png" max-width="750px" caption="Searching in Metrics explorer" >}}
+
+## Redshift and Athena: Async query data support
+
+<!-- Isabella Siu, Kevin Yu, Andrés Martínez -->
+
+_Generally available in all editions of Grafana_
+
+Async query data support in Redshift and Athena makes queries over multiple requests (starting, checking status, and fetching the results) instead of in a single request query. This is useful for queries that can potentially run for a long time and time out. This feature was previously available behind a feature toggle and is now be generally available and enabled by default.
+
+## Redshift and Athena: Async query caching
+
+<!-- Isabella Siu -->
+
+_Experimental in Grafana Enterprise, Cloud Pro, and Cloud Advanced_
+
+This feature adds support for query caching of async queries in the Athena and Redshift data source plugins. To try this feature, enable both the `useCachingService` and `awsAsyncQueryCaching` feature toggles. If you’re using Grafana Cloud and would like to enable this experimental feature, please contact customer support.
+
+### CloudWatch logs Monaco query editor
+
```

Comment:
@iwysiu - Is this section meant to be a child of the **Redshift and Athena: Async query caching item**? Or just an item under Data sources?

**Note, please don't commit any changes without checking in with Quynton first.**

Reply:
It should be an item under data sources